connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -8350,6 +8363,7 @@ object functions {
    *   function name that follows the SQL identifier syntax (can be quoted, can be qualified)
    * @param cols
    *   the expression parameters of function
+   * @group normal_funcs
```

Comment:
How about add a new group `calling_funcs`?

Reply:
we don't have `calling_funcs` after https://github.com/apache/spark/commit/17da43803fd4c405fda00ffc2c7f4ff835ab24aa

because I feel it is more like to `def expr(expr: String): Column` which is in group `normal_funcs`

Reply:
I means to add a new group. If no need, `normal_funcs` is OK too.

Reply:
I personally feel `calling_funcs` is a bit confusing, so let's follow `def expr` 's group